other-charts/prepull-daemonset/templates/daemonset.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ .Chart.Name | trunc 63 | trimSuffix "-" }}
+  labels:
+    app: {{ .Chart.Name | trunc 63 | trimSuffix "-" }}
+spec:
+  selector:
+    matchLabels:
+      name: {{ .Chart.Name | trunc 63 | trimSuffix "-" }}
+  {{- with .Values.updateStrategy }}
+  updateStrategy:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  template:
+    metadata:
+      annotations:
+        {{- if .Values.randomize }}
+        rollme: {{ randAlphaNum 10 | quote }}
+        {{- end }}
+      labels:
+        name: {{ .Chart.Name | trunc 63 | trimSuffix "-" }}
+    spec:
+      initContainers:
+      {{- range .Values.images }}
+        - name: prepull-{{ .name }}
+          image: {{ .image }}
+          imagePullPolicy: Always
```

Comment:
Not necessarily in this PR, but I think some customers will need to set an  `imagePullSecrets` here from an optional `{{ .imagePullSecret }}` in the config

Reply:
Great call! I'll create an issue for follow-up